src/ucp/api/ucp_compat.h

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
hmm...none of the other .h files in src/ucp/api do.  and with the copyright comment enclosed in a doxygen comment block, it can appear in the document.  for example, add ucp_compat.h to the FILE_PATTERNS list in doc/doxygen/ucxdox (to pick up the newly deprecated list) and generate a new ucx.pdf.  when i do that, i see the Mellanox copyright info printed out in the ucp_request_is_completed() module documentation.